api/src/main/java/io/opentelemetry/trace/propagation/TracingContextUtils.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.trace.propagation;
+
+import io.grpc.Context;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.context.propagation.ContextUtils;
+import io.opentelemetry.trace.DefaultSpan;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.SpanContext;
+import io.opentelemetry.trace.Tracer;
+import javax.annotation.concurrent.Immutable;
+
+/**
+ * Util methods/functionality to interact with the {@link io.grpc.Context}.
+ *
+ * <p>Users must interact with the current Context via the public APIs in {@link Tracer} and avoid
+ * accessing this class directly.
+ *
+ * @since 0.1.0
+ */
+@Immutable
+public final class TracingContextUtils {
+  private static final Context.Key<Span> CONTEXT_SPAN_KEY =
+      Context.<Span>key("opentelemetry-trace-span-key");
+  private static final Context.Key<SpanContext> CONTEXT_SPANCONTEXT_KEY =
+      Context.<SpanContext>key("opentelemetry-trace-spancontext-key");
+  private static final Span DEFAULT_SPAN = DefaultSpan.getInvalid();
+
+  /**
+   * Creates a new {@code Context} with the given value set.
```

Comment:
Since the method name changed, I would s/value/span in this and in the next methods' Javadoc